### PR TITLE
Expand check for /it:test workflow to include other author roles

### PR DESCRIPTION
**Pull request by errt**
This should hopefully solve the issue we had recently with the it:test action not running (see #238).

### CODE REVIEW

**Review by johannesduesing — APPROVED** [review]
LGTM - if i remeber correctly from the last time i worked on this, it needs to be merged to develop before i can test it out, right? I'd use #238 to test this after it's merged 👍

**Comment by errt** [comment]
I think so, too.

**Comment by johannesduesing** [comment]
Works as expected 👍

**Pull request merged**
